README.org

```diff
@@ -0,0 +1,112 @@
+#+TITLE: micronota
+
+[[https://coveralls.io/github/biocore/micronota?branch=master][https://coveralls.io/repos/biocore/micronota/badge.svg?branch=master&service=github]]
+[[https://travis-ci.org/biocore/micronota][https://travis-ci.org/biocore/micronota.svg?branch=master]]
+[[https://gitter.im/biocore/micronota?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge][https://badges.gitter.im/Join%20Chat.svg]]
+
+micronota is an open-source, BSD-licensed package to annotate microbial genomes and metagenomes.
+
+As Python 3 matures and majority python packages support Python 3, the scientific Python community is in favor of dropping Python 2 compatibility. Thus, micronota will only support Python 3. This will allow micronota to have few dependency and avoid maintenance of Python 2 legacy code.
+
+* Install
+
+  To install the latest release of micronota:
+  #+BEGIN_SRC sh
+    conda install micronota
+  #+END_SRC
+
+
+  Or you can install through ~pip~:
+  #+BEGIN_SRC sh
+    pip install micronota
+  #+END_SRC
+
+  To install the latest developping version:
+  #+BEGIN_SRC
+    pip install git+git://github.com/biocore/micronota.git
+  #+END_SRC
+
+* Prepare Databases
+
+  To prepare (download and format) the files of TIGRFAM to the right form read by micronota:
+  #+BEGIN_SRC
+    micronota database prepare tigrfam --cache_dir ~/database
+  #+END_SRC
+
+* Config File
+
+  By default, micronota will read =~/.micronota.config= file to set up the environment or tune the parameters, if this config file exists.
+
+  For example, the default directory to store the database files is =~/micronota_db=, but you can override it to =/home/username/db= by setting this in =~/.micronota.config=:
+  #+BEGIN_EXAMPLE
+    [DEFAULT]
+    db_path = /home/username/db
+  #+END_EXAMPLE
+
+  micronota will look for the key =db_path= in the section =DEFAULT= to update the database path.
+
+  Besides setting up the environment, you can also specify the parameter for each individual tools. For example, if you want to run Prodigal with genetic translation table 1, instead of the default translation table, you can create a new file run.cfg (although you can also just add into =~/.micronota.config=)::
```

Comment:
what do you mean weird? it is rendered as literal. do you mean double colons? will fix that.